src/actions/index.js

```diff
@@ -47,244 +47,235 @@ export function receiveAnalysis(api: string, json: Analysis) {
 			datasets: json,
-			receivedAt: Date.now(),
+			receivedAt: Date.now()
 		});
 	};
 }
 
-export const RECEIVE_USAGE = "RECEIVE_USAGE";
-export const RECEIVE_LAB_USAGE = "RECEIVE_LAB_USAGE";
-export function receieveUsage(type, json){
+export const RECEIVE_USAGE = 'RECEIVE_USAGE';
+export const RECEIVE_LAB_USAGE = 'RECEIVE_LAB_USAGE';
+export function receieveUsage(type, json) {
 	return (dispatch: Dispatch) => {
 		dispatch({
 			type: type,
 			usage: json,
-			receivedAt: Date.now(),
+			receivedAt: Date.now()
 		});
 	};
 }
 
-export function deleteItem(type: string, json){
+export function deleteItem(type: string, json) {
 	return (dispatch: Dispatch) => {
 		dispatch({
 			type: type,
 			item: json,
-			receivedAt: Date.now(),
+			receivedAt: Date.now()
 		});
 	};
 }
 
 export function deleteItemById(resourceType, id) {
-	let endpoint = "";
-	if (resourceType === "datasets") {
+	let endpoint = '';
+	if (resourceType === 'datasets') {
 		endpoint = `${config.dataService}/${id}`;
-	}
-	else if (resourceType === "mappings" || resourceType === "fragilities" || resourceType === "restorations"
-		|| resourceType === "repairs") {
+	} else if (
+		resourceType === 'mappings' ||
+		resourceType === 'fragilities' ||
+		resourceType === 'restorations' ||
+		resourceType === 'repairs'
+	) {
 		endpoint = `${config.dfr3ServiceBase}${resourceType}/${id}`;
-	}
-	else if (resourceType === "earthquakes" || resourceType === "tsunamis" || resourceType === "floods"
-		|| resourceType === "tornadoes" || resourceType === "hurricanes" || resourceType === "hurricaneWindfields") {
+	} else if (
+		resourceType === 'earthquakes' ||
+		resourceType === 'tsunamis' ||
+		resourceType === 'floods' ||
+		resourceType === 'tornadoes' ||
+		resourceType === 'hurricanes' ||
+		resourceType === 'hurricaneWindfields'
+	) {
 		endpoint = `${config.hazardServiceBase}${resourceType}/${id}`;
 	}
 	return (dispatch: Dispatch) => {
-		return fetch(endpoint, {mode: "cors", method:"DELETE", headers: getHeader()})
-			.then(response => {
-				if (response.status === 200) {
-					response.json().then(json => {
-						dispatch(deleteItem(DELETE_ITEM, json));
-					});
-				}
-				else if (response.status === 401) {
-					cookies.remove("Authorization");
-					dispatch(deleteItem(LOGIN_ERROR, []));
-				}
-				else {
-					dispatch(deleteItem(DELETE_ERROR, null));
-				}
-			});
+		return fetch(endpoint, { mode: 'cors', method: 'DELETE', headers: getHeader() }).then((response) => {
+			if (response.status === 200) {
+				response.json().then((json) => {
+					dispatch(deleteItem(DELETE_ITEM, json));
+				});
+			} else if (response.status === 401) {
+				cookies.remove('Authorization');
+				dispatch(deleteItem(LOGIN_ERROR, []));
+			} else {
+				dispatch(deleteItem(DELETE_ERROR, null));
+			}
+		});
 	};
 }
 
-export const RECEIVE_HAZARDS = "RECEIVE_HAZARDS";
+export const RECEIVE_HAZARDS = 'RECEIVE_HAZARDS';
 
 export function receiveHazards(type: string, json: Hazards) {
```

Comment:
my eslint complained that only typescript can have types; do you want to change .js to .ts?

Reply:
I was planning to do that eventually as and how we migrate each component, we can clean up actions too and then finally have proper interface definitions that are consistent. I observed that some of the types are not consistent. Editing them now didn't seem a good idea in the PR for the Login component. I would like to know what your thoughts are on this too. :)